src/sentry/utils/performance_issues/detectors/io_main_thread_detector.py

```diff
@@ -224,10 +224,10 @@ def __init__(self, settings: dict[DetectorType, Any], event: dict[str, Any]) ->
         self.mapper = None
         self.parent_to_blocked_span = defaultdict(list)
 
-    def _fingerprint(self, span_list) -> str:
+    def _fingerprint(self, span_list: list[Span]) -> str:
         description_strings = []
         for span in span_list:
-            description_strings.append(span.get("description"))
+            description_strings.append(span.get("description", ""))
```

Comment:
Note to self: `"-".join([None])` fails, so this makes sense. I don't think any SDK actually sends a span that has well-formed main thread data for an IO span but doesn't have a description. A little funny that the code doesn't actually check for this